doc/sphinx/make-your-own-compositor.md

```diff
@@ -0,0 +1,248 @@
+---
+discourse: 4911,5164,5603,6756,8037
+---
```

Comment:
What does this header do nowwadays?

Reply:
It creates this section in the sidebar:
![image](https://github.com/user-attachments/assets/15f463eb-0437-4991-b4d1-bdab37602cd9)

Reply:
@mattkae you can follow the RTD link in the "checks" to see the "live" docs